dev/tools/localization/gen_l10n.dart

```diff
@@ -232,11 +232,37 @@ const Set<String> allowableDateFormats = <String>{
 // * <https://pub.dev/documentation/intl/latest/intl/NumberFormat-class.html>
 const Set<String> allowableNumberFormats = <String>{
   'compact',
+  'compactCurrency',
+  'compactSimpleCurrency',
   'compactLong',
+  'currency',
   'decimalPattern',
   'decimalPercentPattern',
   'percentPattern',
   'scientificPattern',
+  'simpleCurrency',
+};
+
+// A subset of [allowableNumberFormats] that takes named parameters instead
```

Comment:
I think you mean: The names of the NumberFormat factory constructors which have named parameters rather than positional parameters.